shoop/front/middleware.py

```diff
@@ -77,6 +80,9 @@ def _set_shop(self, request):
 
     def _set_person(self, request):
         request.person = get_person_contact(request.user)
+        if not request.person.is_active:
+            messages.add_message(request, messages.INFO, _("Logged out since this account is inactive."))
+            logout(request)
```

Comment:
Hmm.. it seems to happen already. Didn't first see why, but then found out that it's updated by the signal. I'll add a comment, since otherwise this code is hard to understand. Maybe it's still better to explicitly set it to anonymous too, just to be sure (signal might not be connected for some reason).